setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         # Ensure our temporary build directory exists
         try:
             os.makedirs(build_temp)
-        except IOError:
+        except:
```

Comment:
According to the [latest docs](https://docs.python.org/3.5/library/os.html#os.makedirs), an `OSError` is raised if the directory exists, so the code should probably read
```
except OSError:
```
Moreover, `IOError` doesn't exist since Python 3.3; it has been [merged](https://docs.python.org/3.4/library/exceptions.html#OSError) into `OSError`. This is not the case in Python 2.x, where `IOError` and `OSError` are separate exceptions. However, this shouldn't matter, because [`os.makedirs`](https://docs.python.org/2.6/library/os.html#os.makedirs) also raises `OSError` in Python 2.6 and 2.7.

It appears to me that catching an `IOError` is a mistake and catching `OSError` is the correct response, given the code that rests inside the `try` block. However, if there is some peculiar reason for catching `IOError` anyway, one can do so easily:
```
except (OSError, IOError):
```
This should work in Python 2.6+ and Python 3.3+, which is enough for this library's purpose.